content/01.summary.md

```diff
@@ -1,5 +1,10 @@
 ## Summary
-<!-- The summary is a single paragraph no longer than 150 words. An effective summary includes the following elements: (1) a brief background of the question that avoids statements about how a process is not well understood; (2) a description of the results and approaches/model systems framed in the context of their conceptual interest; and (3) an indication of the broader significance of the work. We discourage novelty claims (e.g., use of the word “novel”) because they are overused, tend not to add meaning, and are difficult to verify. Please do not include references in the summary. -->
+<!-- The summary is a single paragraph no longer than 150 words. An effective summary includes the following elements: (1) a brief background of the question that avoids statements about how a process is not well understood; (2) a description of the results and approaches/model systems framed in the context of their conceptual interest; and (3) an indication of the broader significance of the work. We discourage novelty claims (e.g., use of the word “novel”) because they are overused, tend not to add meaning, and are difficult to verify. Please do not include references in the summary. CURRENT WORD COUNT 145 -->
+With pediatric brain and spinal cord tumors being the leading disease-related cause of death in children, urgently identifying novel, curative therapeutic strategies for these tumors remains a critical need.
+To accelerate this process, the Children's Brain Tumor Network and Pacific Pediatric Neuro-Oncology Consortium have created a systematic process for tumor biobanking, model generation, and sequencing with immediate access to harmonized data.
+Here, we create the OpenPBTA project, in which we openly and collaboratively genomically characterize and molecularly subtype the largest cohort of pediatric brain tumor specimens and histologies to date.
+We establish over 40 scalable analysis modules, demonstrate recapitulation of key oncogenic drivers across histologies, and make the code and data openly available.  
```

Comment:
```suggestion
We establish over 40 scalable analysis modules, recapitulate key oncogenic drivers across histologies, and make the code and data openly available.  
```

I think it would be good to add a few more words about the modules, like "over 40 scalable modules leveraging open-source tools to perform common cancer genomics analyses". This is a bit wordy for this sentence, but I think more context about what _are_ these modules would be good.